routes/conversation.js

```diff
@@ -255,6 +257,74 @@ exports.leave = function (req, res){
 	});
 };
 
+/**
+ * Action when a user archives a conversation.
+ */
+exports.archive = function (req, res){
+	if (!req.user || !req.user._id){
+		resError(res, "Access denied.", "/error");
+		return;
+	}
+
+	// Search through the user's conversation list until the conversation with the correct
+	// id is found. 
+	User.findById( req.user._id, function (err, user){
+		if (err){
```

Comment:
No worries, Erik! That seems like a reasonable assumption to make. I'll make the change here